sdk/samples/iot/generate_certificate.ps1

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
if this script needs to be used for testing, it should not be here. We removed it to simplify the user experience and having it here will cause headaches when people ask for support on it.